src/phaser-game/scenes/FortNerf.ts

```diff
@@ -1,168 +1,198 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Perhaps, if you find you need these rules disabled, address in the eslintrc.json instead?